cmd/meroxa/root/connectors/connect_test.go

```diff
@@ -103,45 +103,44 @@ func TestConnectExecution(t *testing.T) {
 		EXPECT().
 		CreateConnector(
 			ctx,
-			meroxa.CreateConnectorInput{
+			&meroxa.CreateConnectorInput{
 				Name:       "",
 				ResourceID: rSource.ID,
 				Configuration: map[string]interface{}{
-					"key":   "value",
-					"input": "my-resource.Table",
-				},
-				Metadata: map[string]interface{}{
-					"mx:connectorType": "source",
+					"key": "value",
 				},
+				Metadata:     map[string]interface{}{},
```

Comment:
How come we're no longer comparing on `Metadata`?

Reply:
The value we put in isn't the totality of the value we get out, so it got messy. there may be a way to adjust for that, i didn't spend a lot of time on it.